include/ada/url-inl.h

```diff
@@ -223,7 +227,7 @@ ada_really_inline size_t url::parse_port(std::string_view view,
     return 0;
   }
   ada_log("parse_port: ", parsed_port);
-  const size_t consumed = size_t(r.ptr - view.data());
+  auto const consumed = static_cast<size_t>(r.ptr - view.data());
```

Comment:
Why? You went from

```C++
const size_t consumed = size_t(r.ptr - view.data());
```

to...

```
auto const consumed = static_cast<size_t>(r.ptr - view.data());
```

I'd go in the other direction...

```C++
size_t consumed(r.ptr - view.data());
```

Less code, more readable.

Reply:
Intent is more important than readability. We are "casting". implicit casting is a waste of time if we wanted to fix anything.

We SHOW the intent by using `static_cast`, and use `auto` to not repeat ourselves.